data/systemd/dbus-org.learningequality.Kolibri.Daemon.service.in

```diff
@@ -6,5 +6,6 @@ ConditionPathExists=/var/lib/flatpak/app/@KOLIBRI_FLATPAK_ID@
 Type=dbus
 BusName=@KOLIBRI_DAEMON_SERVICE@
 ExecStart=@libexecdir@/@KOLIBRI_CMD_PREFIX@-daemon
+KillMode=mixed
```

Comment:
This sounds important!

Place the systemd kill document https://www.freedesktop.org/software/systemd/man/latest/systemd.kill.html